skills/findChannel.js

```diff
@@ -0,0 +1,21 @@
+class Project{
+  constructor(channel){
+    this.channel_id = channel;
+  }
+
+  static findByChannel(channel){
+    return new Promise(function(resolve,reject){
+      let p = new Project(channel)
+    // p.channel_id = channel;
+      resolve(p);
+    });
+
+  }
+
+}
+
+async function getNewProject(x){
+  let project = await Project.findByChannel(x);
+  console.log(project)
+}
+getNewProject('c5678g')
```

Comment:
Looks like you might be locally testing by running this file through node. The class you’ve defined here should be consumed as an `import`. Let’s remove this debugging code and export our class.

👏

Reply:
Okay, will do.